### PR TITLE
Add responsive hamburger menu for mobile view

### PR DESCRIPTION
**Pull request by tims139**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -20,10 +21,39 @@ function Navigation() {
...
+          ) : (
+            <Bars3Icon aria-hidden="true" className="relative h-6 w-6" />
+          )}
+        </button>
       </div>
+
+      {/* Dropdown menu */}
+      {menuOpen && (
+        <div className="absolute top-0 right-0 bg-white shadow-lg pr-9 z-40 md:hidden">
+          <ul className="flex flex-col p-4">
```
Comment (akhilwaghmare) [inline src/App.jsx]: Styling suggestion: Add a top padding so that the links start below the x icon

File: src/App.jsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import LogIn from "./pages/LogIn";
 
 function Navigation() {
```
Comment (akhilwaghmare) [inline src/App.jsx]: For organization, think about moving the `Navigation` component into its own file.

**Review by akhilwaghmare — CHANGES_REQUESTED** [review]
Overall looking good, requested a quality of life changes and styling suggestion

File: src/App.jsx (hunk 1 above)
Comment (tims139) [reply src/App.jsx]: Done!

File: src/App.jsx (hunk 2 above)
Comment (tims139) [reply src/App.jsx]: Done!

**Review by akhilwaghmare — APPROVED** [review]

**Pull request merged**